core/corehttp/gateway.go

```diff
@@ -24,18 +24,13 @@ import (
 	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
 )
 
-func GatewayOption(writable bool, paths ...string) ServeOption {
+func GatewayOption(paths ...string) ServeOption {
 	return func(n *core.IpfsNode, _ net.Listener, mux *http.ServeMux) (*http.ServeMux, error) {
 		cfg, err := n.Repo.Config()
 		if err != nil {
 			return nil, err
 		}
 
-		api, err := coreapi.NewCoreAPI(n, options.Api.FetchBlocks(!cfg.Gateway.NoFetch))
-		if err != nil {
-			return nil, err
-		}
-
```

Comment:
@hacdias why we removed this? `Gateway.NoFetch` is not related to writable gateway, it controls if gateway should fetch content that is not already in local datastore.

Reply:
Removed because this was only used for the writable gateway